src/main/scala/nl/knaw/dans/easy/multiDeposit/CommandLineOptions.scala

```diff
@@ -13,68 +13,83 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package nl.knaw.dans.easy.multiDeposit
+package nl.knaw.dans.easy.multideposit
 
 import java.io.File
 
 import com.typesafe.config.{Config, ConfigFactory}
-import org.rogach.scallop.ScallopConf
+import nl.knaw.dans.easy.multideposit.CommandLineOptions._
+import org.rogach.scallop.{ScallopConf, singleArgConverter}
 import org.slf4j.LoggerFactory
 
-import scala.util.Properties
-
 object CommandLineOptions {
 
   val log = LoggerFactory.getLogger(getClass)
-  val mdInstructionsFileName = "instructions.csv"
 
   def parse(args: Array[String]): Settings = {
     log.debug("Loading application.conf ...")
-    val conf = ConfigFactory.load
+    lazy val conf = ConfigFactory.load
```

Comment:
Hadn't done this in SplitMultiDeposit yet. Was looking at IngestFlow this morning. Maybe a better idea to do this for all projects at once?